Content.Server/Nyanotrasen/Item/PseudoItem/PseudoItemSystem.cs

```diff
@@ -122,19 +128,19 @@ public bool TryInsert(EntityUid storageUid, EntityUid toInsert, PseudoItemCompon
             return false;
 
         var item = EnsureComp<ItemComponent>(toInsert);
+        _tagSystem.TryAddTag(toInsert, PreventTag);
```

Comment:
Why is PseudoItem the one adding the tag?

Reply:
Because it'd apply to anything else that is a pseudoitem aswell. Non-items shouldn't be labeled, and as pseudoitem adds an itemcomponent to allow inserting into storage, stuff that shouldn't be labeled can be labeled

Reply:
Alright